### PR TITLE
STCOM-809 Implement warning callout

### PR DESCRIPTION
**Pull request by axelhunn**
https://issues.folio.org/browse/STCOM-809
Adding Callout type `warning` , like in MessageBanner, same style

image

### CODE REVIEW

**Review by zburke — COMMENTED** [review]
The code you added looks good, but can you please explain why all the `.base` style information was removed? The amount of code you added is dwarfed by what you removed. If it was unused cruft or something, that's fine, just please explain what's going on.

**Review by doytch — APPROVED** [review]

**Comment by axelhunn** [comment]
> The code you added looks good, but can you please explain why all the `.base` style information was removed? The amount of code you added is dwarfed by what you removed. If it was unused cruft or something, that's fine, just please explain what's going on.

I think it's unused, yes. Changing in doesn't change anything. I tried search usage of `base` in the code - seems nothinng

File: lib/Callout/Callout.css [hunk 1]
```diff
@@ -88,6 +56,11 @@
     background-color: color(var(--success, #060) tint(75%));
```
Comment (axelhunn) [inline lib/Callout/Callout.css:56]: I'm not sure about this `color` function
Does it work anywhere except Safari?
https://caniuse.com/css-color-function

image

Comment (axelhunn) [reply lib/Callout/Callout.css:56]: Maybe just use `--success-fill` here?

Comment (axelhunn) [reply lib/Callout/Callout.css:56]: Or maybe the opacity in `-fill` colors is too much for the Callouts
image

**Review by ViktorSoroka07 — APPROVED** [review]

**Review by NikitaSedyx — APPROVED** [review]

File: lib/Callout/Callout.css (hunk 1 above)
Comment (axelhunn) [reply lib/Callout/Callout.css:56]: OK, for the `warning` I make the color function, the same as in other types.
I have no idea for now, why it does work in production, but does not in the storybook.

**Pull request merged**